packages/app/src/sandbox/compile.ts

```diff
@@ -420,7 +421,7 @@ async function compile({
     const templateDefinition = getDefinition(template);
     const configurations = parseConfigurations(
       template,
-      templateDefinition.configurationFiles,
+      omit(templateDefinition.configurationFiles, '/netlify.toml'),
```

Comment:
Does this break if `omit` is removed? It feels like this doesn't scale really well when we have more configs that are not `json`.

Reply:
Fixed!